packages/components/src/BrandMoment/_docs/BrandMoment.stories.tsx

```diff
@@ -1,102 +1,132 @@
 /* eslint-disable jsx-a11y/anchor-is-valid */
 import React from "react"
-import { Meta, StoryFn } from "@storybook/react"
-import { BrandMoment } from "@kaizen/brand-moment"
+import { Meta, StoryObj } from "@storybook/react"
+import isChromatic from "chromatic"
 import {
   BrandMomentCaptureIntro,
   BrandMomentPositiveOutro,
   BrandMomentError,
+  AnimatedSceneProps,
 } from "@kaizen/draft-illustration"
 import { Paragraph } from "@kaizen/typography"
-import arrowLeftIcon from "../../../icons/arrow-left.icon.svg"
-import arrowRightIcon from "../../../icons/arrow-right.icon.svg"
-import mailIcon from "../../../icons/email.icon.svg"
-import feedbackClassifyIcon from "../../../icons/feedback-classify.icon.svg"
-import securityTipIcon from "../../../icons/security-tip.icon.svg"
+// import { ArrowLeftIcon } from "~icons/ArrowLeftIcon"
+import { ArrowRightIcon } from "~icons/ArrowRightIcon"
+import { EmailIcon } from "~icons/EmailIcon"
+// import feedbackClassifyIcon from "../../../icons/feedback-classify.icon.svg"
+// import securityTipIcon from "../../../icons/security-tip.icon.svg"
+import { BrandMoment } from "../index"
 import {
   MinimalBasic,
   MinimalCustomerFocused,
   FakeNavBar,
 } from "./ExampleHeaders"
 
-export default {
+const IS_CHROMATIC = isChromatic()
+
+const illustrationProps = (
+  IS_CHROMATIC ? {} : { isAnimated: true, loop: true }
+) satisfies AnimatedSceneProps
+
+const meta = {
   // Not to be released until full KAIO migration
   // tags: ["autodocs"],
   title: "Components/Brand Moment",
   component: BrandMoment,
+  parameters: {
+    chromatic: { disable: false },
+  },
+} satisfies Meta<typeof BrandMoment>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+const IconRTLTemplate: Pick<Story, "render" | "parameters"> = {
+  render: ({ primaryAction, ...args }, { globals }) => (
+    <BrandMoment
+      {...args}
+      primaryAction={{
+        label: "Primary action label",
+        ...primaryAction,
+        icon:
+          globals.textDirection === "ltr" ? (
+            <ArrowRightIcon />
```

Comment:
Naughty 😛 

Interesting that this made it past axe, maybe because it's put inside a button